docs/src/howto-guides/howto-guides.md

```diff
@@ -4,8 +4,8 @@
 2. [How to create a test certificate?](./003_registration.md)
 3. [How to connect a cloud end-point?](./004_connect.md)
 4. [How to use `tedge mqtt` module?](./005_pub_sub.md)
-5. [How to test connection to cloud?](./007_test_connection.md)
-6. [How to test the cloud connection?](./007_test_connection.md)
+5. [How to configure thin-edge.io?](./006_config.md)
+6. [How to test connection to cloud?](./007_test_connection.md)
```

Comment:
Thanks for spotting that a link was duplicated.

However, I prefer the second wording.

```suggestion
6. [How to test the cloud connection?](./007_test_connection.md)
```